src/elliptic.jl

```diff
@@ -0,0 +1,234 @@
+# References:
+#  [1] Abramowitz, Stegun: Handbook of Mathematical Functions (1965)
```

Comment:
While A&S is great, would it be possible to link to DLMF as well? It has the advantage of being able to refer to equations via URLs, e.g. https://dlmf.nist.gov/7.12.1

Reply:
I'll have a look tomorrow whether I find the formulae in DLMF as well, and will add references if possible.

Reply:
Updated. DLMF is great, thanks for pointing this out! Could have saved me from lugging a copy of A&S with me for a week... :smile:

Reply:
It is really useful, particularly for this sort of stuff. As I understand, it is intended to be the successor to A&S